tmt/steps/__init__.py

```diff
@@ -167,6 +173,12 @@ class (single class or tuple of classes).
                 if classes is None or isinstance(plugin, classes)],
             key=lambda plugin: plugin.order)
 
+    def try_reboot(self):
+        """ Run all loaded Reboot plugin instances of the step """
+        for plugin in self.plugins():
+            if isinstance(plugin, Reboot):
+                plugin.go()
+
```

Comment:
If `try_reboot` and `try_running_log` are always called together (when the step has been already done) would it make sense to have a single `actions()` method to cover both `Login` and `Reboot`?

Reply:
I hope, it's solved now.

Reply:
Looks good.